src/Sulu/Component/Content/Document/Subscriber/OrderSubscriber.php

```diff
@@ -64,19 +73,18 @@ public function handlePersist(PersistEvent $event)
 
         $propertyName = $this->encoder->systemName(self::FIELD);
 
-        if ($node->hasProperty($propertyName)) {
-            return;
-        }
-
         $parent = $node->getParent();
-        $nodeCount = count($parent->getNodes());
-        $order = ($nodeCount + 1) * 10;
+        $count = 0;
+        foreach ($parent->getNodes() as $childNode) {
```

Comment:
@dantleech I am missing the node in the reorder event... Is this something that should be added in the `RegistratorSubscriber`? I would have handled it the same way as `handleNodeFromRegistry`, but just with the `ReorderEvent` (which has to be extended by a node also, so is it a good idea to let it inherit from the `AbstractMappingEvent`?).